vignettes/release_strategy.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Release Strategy"
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+    toc_depth: 6
+vignette: >
+  %\VignetteIndexEntry{Release Strategy}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+This article explains how we do package releases for `{admiral}` and across all our
+package extensions. This includes details around planned/scheduled releases, as well
+as hotfixes.
+
+# Release Schedule
+
+A release schedule is maintained on the [homepage](../index.html#release-schedule).
```

Comment:
Should this get linked to the admiral core release schedule.  https://pharmaverse.github.io/admiral/#release-schedule

Reply:
maybe - i spotted this too but left it as it was what was there before, and the link takes you to the admiraldev section which in turns links to the admiral section. want me to update it? is there any worry about circular dependencies somehow here, as admiraldev comes first and then admiral - i guess not as its just a link, but just thinking out loud... anyway, let me know which you prefer!

Reply:
I think it is nicer for direct link then having to click a few more times to get to it?

Reply:
oohhh i remember why i coded it like that.  the release was on `devel` branch on admiral core so I couldn't link anything at that time